.ci/buildchain-config.yaml

```diff
@@ -25,7 +25,7 @@ default:
       mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }}
     after:
       current: |
-        docker system prune -f
+        bash -c "docker system prune -f 2>&1 | grep -q 'a prune operation is already running' && echo 'Prune operation already in progress. Skipping this command.' || true"
```

Comment:
maybe better to use different command than `prune`?

see this for instance https://github.com/docker/for-mac/issues/2501
As recommended `docker rmi $(docker images -f "dangling=true"-q)` is logging the progress